hw-model/test-fw/mailbox_responder.rs

```diff
@@ -0,0 +1,63 @@
+// Licensed under the Apache-2.0 license
+
+//! A very simple program that responds to the mailbox.
+
+#![no_main]
+#![no_std]
+
+// Needed to bring in startup code
+#[allow(unused)]
+use caliptra_test_harness;
+
+use caliptra_registers;
+
+#[panic_handler]
+pub fn panic(_info: &core::panic::PanicInfo) -> ! {
+    loop {}
+}
+
+#[no_mangle]
+extern "C" fn main() {
+    let soc_ifc = caliptra_registers::soc_ifc::RegisterBlock::soc_ifc_reg();
+    soc_ifc.cptra_flow_status().write(|w| w.ready_for_fw(true));
+    let mbox = caliptra_registers::mbox::RegisterBlock::mbox_csr();
+
+    loop {
+        while !mbox.status().read().mbox_fsm_ps().mbox_execute_uc() {}
+        let cmd = mbox.cmd().read();
+
+        match cmd {
+            0x1000_0000 => {
```

Comment:
nit: Would be helpful for understanding the tests to add comments explaining what each command is supposed to do

Reply:
Done